app/main/posts/common/collection-toggle/collection-toggle-button.html

```diff
@@ -2,5 +2,5 @@
     <svg class="iconic">
         <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/img/iconic-sprite.svg#grid-three-up"></use>
     </svg>
-    <span class="button-label" translate="set.add_to_collection">Add to collection</span>
+    <span class="button-label hidden" translate="set.add_to_collection">Add to collection</span>
```

Comment:
I'm not sure about this. Why is this being hidden ?

Reply:
Seth had me add it for accessibility reasons.